variables.tf

```
@@ -210,6 +210,12 @@ variable "node_use_max_pods" {
   description = "Set to false to prevent EKS from setting --max-pods in Kubelet config. By default, EKS sets the maximum number of pods that can run on the node, based on the instance type. Disabling this can be useful when using a CNI other than the default, like Calico."
 }
 
+variable "show_max_allowed" {
+  type        = bool
+  default     = true
+  description = "Set to false to prevent setting show_max_allowed on max-pods-calculator"
```

It should probably not be the default, and could be better described : 
```suggestion
  default     = false
  description = "Set to show max number of pods allowed to run in node instead of recommended value"
```